docs/hooks/hooks.md

```diff
@@ -28,9 +28,15 @@ const externalContracts = {
       address: "0x...",
       abi: [...],
     },
-    USDT: {
+    WETH: {
       address: "0x...",
       abi: [...],
     },
```

Comment:
I think in the example we had here there were missing closing brace for each chainId. Afraid the syntax error was in the SE-2 code example too, will create a PR.

Here is the code with the missing closing braces:
```tsx
const externalContracts = {
  1: {
    DAI: {
      address: "0x...",
      abi: [...],
    },
    WETH: {
      address: "0x...",
      abi: [...],
    },
  },
  5: {
    DAI: {
      address: "0x...",
      abi: [...],
    },
    WETH: {
      address: "0x...",
      abi: [...],
    },
  },
} as const;
```

Reply:
Tysm !! updated at [568917e](https://github.com/scaffold-eth/se2-docs/pull/45/commits/568917ebe40c883c76678307c8f28b1c3c70d5d5) 🙌